src/components/structures/TimelinePanel.js

```diff
@@ -588,8 +588,8 @@ var TimelinePanel = React.createClass({
                 };
             }
             Modal.createDialog(ErrorDialog, {
-                title: "Failed to load event",
-                description: msg,
+                title: "Failed to load timeline position",
+                description: "Vector was trying to load a specific point in this room's timeline but was unable to find it.",
```

Comment:
As far as I can see, the context API never returns 403, although I might have missed somewhere. Would you expect that to be if you don't have permission to view the event in question?